specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-07-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSet_Create_WithHighSpeedInterconnectPlacement.json

```diff
@@ -0,0 +1,223 @@
+{
+  "parameters": {
```

Comment:
nit: do we need a complete new file for the example? Or we can just add the new property as a part of an existing example file?
What is the model generally used for each new property? Please feel free to close if we follow the model of new file per property